src/lib/chips/chip.ts

```diff
@@ -145,6 +246,7 @@ export class MdChip implements Focusable, OnInit, OnDestroy {
     if (el.nodeName.toLowerCase() == 'mat-basic-chip' || el.hasAttribute('mat-basic-chip') ||
         el.nodeName.toLowerCase() == 'md-basic-chip' || el.hasAttribute('md-basic-chip')) {
       el.classList.add('mat-basic-chip');
+      this._renderer.setElementClass(this._elementRef.nativeElement, `md-chip`, true);
```

Comment:
Is this correct? Do we really need to have the `md-chip` class on the host-element?

And also above with `classList.add('mat-chip')`, I think this is obsolete since we already set the class in the host metadata.